qbft/spectest/tests/controller/latemsg/late_prepare_past_instance.go

```diff
@@ -10,10 +10,19 @@ import (
 
 // LatePreparePastInstance tests process prepare msg for a previously decided instance
 func LatePreparePastInstance() *tests.ControllerSpecTest {
-	identifier := types.NewMsgID(testingutils.TestingValidatorPubKey[:], types.BNRoleAttester)
 	ks := testingutils.Testing4SharesSet()
 
-	allMsgs := testingutils.DecidingMsgsForHeight([]byte{1, 2, 3, 4}, identifier[:], 5, ks)
+	allMsgs := []*qbft.SignedMessage{
```

Comment:
this is kind of a misleading test but DecidingMsgsForHeight takes as input how many decided instance to output. 
The test asks for 5 decided output and then selects message from it to simulate late prepare. 

We better just refactor this to have the actual messages

Reply:
I changed it to use `DecidingMsgsForHeight`. Thanks